### PR TITLE
Animal flavor text

### PR DESCRIPTION
**Pull request by DmitriyRubetskoy**
Hopefully witty enough, and aren't too long. This time only the necessary files, no more glass beakers

### CODE REVIEW

File: Resources/Prototypes/Entities/Mobs/NPCs/animals.yml [hunk 1]
```diff
@@ -482,7 +485,7 @@
   name: penguin
   parent: MonkeyMob_Content
   id: PenguinMob_Content
-  description:
+  description: Their lives are constant pain due to their inner-body knees
```
Comment (SweptWasTaken) [inline Resources/Prototypes/Entities/Mobs/NPCs/animals.yml:485]: Period.

File: Resources/Prototypes/Entities/Mobs/NPCs/animals.yml [hunk 2]
```diff
@@ -225,7 +226,7 @@
   name: bee
   parent: MonkeyMob_Content
   id: BeeMob_Content
-  description:
+  description: Nice to have, but you can't build a civilization on foundation of honey alone
```
[the remark this answers is not in the document]
Comment (DmitriyRubetskoy) [reply Resources/Prototypes/Entities/Mobs/NPCs/animals.yml:228]: It is a reference to the https://www.youtube.com/watch?v=JEYh5WACqEk&t=518s on 9:04

File: Resources/Prototypes/Entities/Mobs/NPCs/animals.yml [hunk 3]
```diff
@@ -139,6 +139,7 @@
   name: butterfly
   parent: MonkeyMob_Content
   id: ButterflyMob_Content
+  description: Despite popular misconseptions, is not actually made of butter
```
[the remark this answers is not in the document]
Comment (Injazz) [reply Resources/Prototypes/Entities/Mobs/NPCs/animals.yml]: I don't feel "popular misconceptions" is appropriate wording.
Use either _"popular beliefs"_, or_"common misconceptions"_.
Also there is a typo in "miscon**s**eptions"

**Review by Injazz — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Entities/Mobs/NPCs/animals.yml [hunk 4]
```diff
@@ -447,7 +447,7 @@
   name: crab
   parent: MonkeyMob_Content
   id: CrabMob_Content
-  description:
+  description: A folk legend goes around that his claw snaps spacemen out of existance over distasteful remarks. Be polite and tolerant for your own safety.
```
Comment (Injazz) [inline Resources/Prototypes/Entities/Mobs/NPCs/animals.yml:450]: existence*

**Comment by DmitriyRubetskoy** [comment]
Damn I need to spellcheck when writing large amounts of text, this is embarrassing

**Review by Zumorica — CHANGES_REQUESTED** [review]

**Review by SweptWasTaken — APPROVED** [review]

**Comment by SweptWasTaken** [comment]
Didn't cringe at any of the descriptions and I think the changes made were constructive. 

Ready for a merge imo.

**Comment by DmitriyRubetskoy** [comment]
Yeah probably good for a merge

**Review by Zumorica — APPROVED** [review]

**Comment by DmitriyRubetskoy** [comment]
Please merge?..

File: Resources/Prototypes/Entities/Mobs/NPCs/animals.yml [hunk 5]
```diff
@@ -225,7 +225,7 @@
   name: bee
   parent: MonkeyMob_Content
   id: BeeMob_Content
-  description:
+  description: Nice to have, but you can't build a civilization on foundation of honey alone.
```
Comment (metalgearsloth) [inline Resources/Prototypes/Entities/Mobs/NPCs/animals.yml:228]:
```suggestion
  description: Nice to have, but you can't build a civilization on a foundation of honey alone.
```

The actual quote.

**Review by metalgearsloth — APPROVED** [review]

**Comment by SweptWasTaken** [comment]
Merge me.

**Review by Zumorica — APPROVED** [review]

**Pull request merged**